comfy_cli/cmdline.py

```
@@ -44,6 +44,33 @@
 from comfy_cli.command.models import models as models_command
 from comfy_cli.command.models import search as models_search_command
 from comfy_cli.config_manager import ConfigManager
+
+def _get_backend_from_env():
+    """Resolve host/port from environment variables as requested.
+    - COMFY_BACKEND=host:port or COMFY_HOST_PORT=host:port (combined, colon-separated)
+    - COMFY_HOST and COMFY_PORT (separate)
+    CLI flags take precedence; this is a fallback.
+    """
+    import os
```

Pylint (4.0.6)

[warning] Redefining name 'os' from outer scope (line 2)

(W0621)